forest/cli/Cargo.toml

```diff
@@ -0,0 +1,91 @@
+forest_auth.workspace            = true
+forest_beacon.workspace          = true
+forest_blocks.workspace          = true
+forest_chain.workspace           = true
+forest_chain_sync.workspace      = true
+forest_cli_shared.workspace      = true
+forest_crypto.workspace          = true
+forest_db.workspace              = true
+forest_encoding.workspace        = true
+forest_fil_types.workspace       = true
+forest_interpreter.workspace     = true
+forest_ipld_blockstore.workspace = true
+forest_json.workspace            = true
+forest_key_management.workspace  = true
+forest_libp2p.workspace          = true
+forest_message                   = { workspace = true, features = ["blst"] }
+forest_message_pool.workspace    = true
+forest_networks.workspace        = true
+forest_paramfetch.workspace      = true
+forest_rpc-api.workspace         = true
+forest_rpc-client.workspace      = true
+forest_state_manager.workspace   = true
+forest_utils.workspace           = true
+futures.workspace                = true
+fvm.workspace                    = true
+fvm_ipld_car.workspace           = true
+fvm_ipld_encoding.workspace      = true
+fvm_shared                       = { workspace = true, default-features = false }
+git-version                      = "0.3"
+hex.workspace                    = true
+lazy_static.workspace            = true
+libp2p                           = { workspace = true, default-features = false, features = ["identify"] }
+log.workspace                    = true
+multibase.workspace              = true
+num-rational.workspace           = true
+num-traits.workspace             = true
+once_cell.workspace              = true
+pbr.workspace                    = true
+pretty_env_logger.workspace      = true
+prometheus                       = { workspace = true, features = ["process"] }
+rayon.workspace                  = true
+reqwest                          = { version = "0.11", features = ["stream"] }
+rpassword                        = "7.0"
+rug                              = "1.17"
+rust-s3                          = "0.32.3"
+serde                            = { workspace = true, features = ["derive"] }
+serde_json.workspace             = true
+sha2                             = "0.10.5"
+strfmt                           = "0.2.2"
+structopt                        = "0.3"
+tempfile                         = "3.3"
+ticker                           = "0.1"
+time                             = "0.3"
+tokio                            = { workspace = true, features = ["sync"] }
+toml                             = "0.5"
+uuid                             = { version = "1.1", features = ["v4"] }
+
+
+[dependencies.jsonrpc-v2]
+default-features = false
+features         = ["easy-errors", "macros", "bytes-v05"]
+version          = "0.11"
+
+[dev-dependencies]
+assert_cmd        = "2"
+quickcheck        = "1"
+quickcheck_macros = "1"
+rand.workspace    = true
+
+[features]
+default = []
+testnet = []
+release = []
```

Comment:
I'm not sure, do we need those?

Reply:
This original build script uses `release` feature in both readme and CI script
```
- name: Cargo Build
        uses: actions-rs/cargo@v1
        with:
          command: build
          args: --release --bin forest --bin forest-cli --features release
```

Reply:
Are those features used anywhere? If not, it'd be nice to get rid of them, perhaps those are some leftovers.

Reply:
They look unused. I've added them to the list in: https://github.com/ChainSafe/forest/issues/2008

Reply:
No neither `release` and `testnet` exist in .rs files

Reply:
fixed

Reply:
Thanks!